desktop/ui/src/main/java/org/datacleaner/util/RemoteServersUtils.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.util;
+
+import java.util.List;
+
+import org.datacleaner.configuration.DataCleanerConfiguration;
+import org.datacleaner.configuration.RemoteServerData;
+import org.datacleaner.configuration.RemoteServerDataImpl;
+import org.datacleaner.descriptors.CompositeDescriptorProvider;
+import org.datacleaner.descriptors.RemoteDescriptorProvider;
+import org.datacleaner.descriptors.RemoteDescriptorProviderImpl;
+import org.datacleaner.restclient.ComponentRESTClient;
+
+/**
+ * Utilities for better work with remote servers.
+ */
+public class RemoteServersUtils {
+    final private DataCleanerConfiguration _configuration;
+
+    public RemoteServersUtils(final DataCleanerConfiguration configuration) {
+        _configuration = configuration;
+    }
+
+    public RemoteServerData getServerConfig(String serverName) {
```

Comment:
This method should be added on ```RemoteServerConfiguration``` in my opinion - to allow general retrieval of server data based on name.